aviatrix-controller-build/outputs.tf

```diff
@@ -1,10 +1,10 @@
 output private_ip {
   value       = length(aws_instance.aviatrixcontroller) > 0 ? aws_instance.aviatrixcontroller.*.private_ip[0] : ""
-  description = "Private IPs for first aviatrix conroller"
+  description = "Private IP for the first aviatrix conroller"
 }
 
 output public_ip {
-  value       = length(aws_instance.aviatrixcontroller) > 0 ? aws_instance.aviatrixcontroller.*.public_ip[0] : ""
+  value       = length(aws_eip.controller_eip) > 0 ? aws_eip.controller_eip.*.public_ip[0] : ""
```

Comment:
in the case where EIP gets detached, the original code would display the IP accurately

Reply:
The original code does not output public_ip.